test/FrozenClockTest.php

```diff
@@ -23,4 +23,22 @@ public function nowShouldReturnAlwaysTheSameObject()
         self::assertSame($now, $clock->now());
         self::assertSame($now, $clock->now());
     }
+
+    /**
+     * @test
+     *
+     * @covers \Lcobucci\Clock\FrozenClock::setTo
+     * @covers \Lcobucci\Clock\FrozenClock::now
```

Comment:
`\Lcobucci\Clock\FrozenClock::now` should be marked as `@uses` instead of `@covers`.

Reply:
We also need to add `@uses \Lcobucci\Clock\FrozenClock::__construct`

Reply:
Fixed